flake.nix

```diff
@@ -36,6 +36,14 @@
       ghc8107  = pkgs.haskell.packages.ghc8107;
       ghc921  = pkgs.haskell.packages.ghc921;
 
+      pythonDevEnv = pkgs.python3.withPackages (p: [
+        p.ipykernel
```

Comment:
All of these dependencies except `jupyterlab` are unnecessary: https://github.com/gibiansky/IHaskell/blob/6362b9b1d68e347749792c783b6bc12a33600d07/release.nix#L41